truffle/src/main/java/org/jruby/truffle/runtime/RubyContext.java

```diff
@@ -309,6 +311,10 @@ public RubyString makeString(RubyClass stringClass, char string, Encoding encodi
         return makeString(stringClass, Character.toString(string), encoding);
     }
 
+    public RubyString makeString(byte[] bytes) {
```

Comment:
The number of these `makeString` cases is getting a bit high.  We may want to rethink where they live.